From 5a3af0dd7ab1eb78ab3933c580837018a880bb8b Mon Sep 17 00:00:00 2001
From: Tim Clephas <tim.clephas@nobleo.nl>
Date: Tue, 11 May 2021 10:35:34 +0200
Subject: [PATCH 1/4] Build with ssl

---
 debian/rules.em | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/debian/rules.em b/debian/rules.em
index 3342408..11efbd2 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -21,6 +21,9 @@ export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG
 # Solve shlibdeps errors in REP136 packages that use GNUInstallDirs:
 export DEB_HOST_MULTIARCH := $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 
+# Build with ssl
+DEB_CXXFLAGS_MAINT_APPEND += -DPAHO_WITH_SSL=ON
+
 %:
 	dh $@@ -v --buildsystem=cmake
 
-- 
2.25.1

