From 659f293d0f4f1b8862f728f918d8b8f39da70313 Mon Sep 17 00:00:00 2001
From: Tim Clephas <tim.clephas@nobleo.nl>
Date: Fri, 28 May 2021 16:03:32 +0200
Subject: [PATCH 3/3] Disable testing as it depends on network connections

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 11efbd2..820a3bc 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -22,7 +22,7 @@ export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG
 export DEB_HOST_MULTIARCH := $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 
 # Build with ssl
-DEB_CXXFLAGS_MAINT_APPEND += -DPAHO_WITH_SSL=ON
+DEB_CXXFLAGS_MAINT_APPEND += -DPAHO_WITH_SSL=ON -PAHO_ENABLE_TESTING=OFF
 
 %:
 	dh $@@ -v --buildsystem=cmake
-- 
2.25.1

