From da0b4984e488e4e24928758992c3bce33cf6c692 Mon Sep 17 00:00:00 2001
From: Tim Clephas <tim.clephas@nobleo.nl>
Date: Fri, 28 May 2021 16:03:32 +0200
Subject: [PATCH 3/5] Disable testing as it depends on network connections

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 4a6f8e3..3a8fcfa 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -25,7 +25,7 @@ endif
 export DEB_HOST_MULTIARCH := $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 
 # Build with ssl
-DEB_CXXFLAGS_MAINT_APPEND += -DPAHO_WITH_SSL=ON
+DEB_CXXFLAGS_MAINT_APPEND += -DPAHO_WITH_SSL=ON -PAHO_ENABLE_TESTING=OFF
 
 DEB_HOST_GNU_TYPE ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
 
-- 
2.34.1

