From 4fdcb58ac45367590882e82e699dd16d8f569423 Mon Sep 17 00:00:00 2001
From: Tim Clephas <tim.clephas@nobleo.nl>
Date: Mon, 31 May 2021 09:38:19 +0200
Subject: [PATCH 4/5] Syntax error

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 3a8fcfa..c987a5b 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -25,7 +25,7 @@ endif
 export DEB_HOST_MULTIARCH := $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 
 # Build with ssl
-DEB_CXXFLAGS_MAINT_APPEND += -DPAHO_WITH_SSL=ON -PAHO_ENABLE_TESTING=OFF
+DEB_CXXFLAGS_MAINT_APPEND += -DPAHO_WITH_SSL=ON -DPAHO_ENABLE_TESTING=OFF
 
 DEB_HOST_GNU_TYPE ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
 
-- 
2.34.1

