From 23cca9eeba1f3f786417c1825071a194a323e26d Mon Sep 17 00:00:00 2001
From: Tim Clephas <tim.clephas@nobleo.nl>
Date: Wed, 2 Jun 2021 09:16:23 +0200
Subject: [PATCH 5/5] They are C flags for the c package

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index c987a5b..88ee9e4 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -25,7 +25,7 @@ endif
 export DEB_HOST_MULTIARCH := $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 
 # Build with ssl
-DEB_CXXFLAGS_MAINT_APPEND += -DPAHO_WITH_SSL=ON -DPAHO_ENABLE_TESTING=OFF
+export DEB_CFLAGS_MAINT_APPEND = -DPAHO_WITH_SSL=ON -DPAHO_ENABLE_TESTING=OFF
 
 DEB_HOST_GNU_TYPE ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
 
-- 
2.25.1

